src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -4973,5 +4976,29 @@ partial class C
             Assert.Equal("SourceFile(Program.cs[52..53))", defSymbol.Locations.Single().ToString());
             Assert.Equal("SourceFile(Program.cs[97..98))", implSymbol.Locations.Single().ToString());
         }
+        
+        [Fact]
+        public void OnlyOneAccessorHasBodyOnImplementation()
+        {
+            var source = """
+                partial class C
+                {
+                    public partial int Prop1 { get; set; }
+                    public partial int Prop1 { get => 1; set; }
```

Comment:
Note that we already have coverage for when a partial property has no accessors with bodies. In that case, the language considers it to be a definition part, and we don't want to report errors on the missing bodies.